src/System Application/App/Extension Management/src/ExtensionDeploymentStatus.Page.al

```diff
@@ -99,6 +100,49 @@ page 2508 "Extension Deployment Status"
                     ExtnDeploymentStatusDetail.Run();
                 end;
             }
+            action(Refresh)
+            {
+                ApplicationArea = All;
+                ToolTip = 'Refresh the deployment details.';
+                Image = Refresh;
+
+                trigger OnAction()
+                var
+                    NavAppTenantOperationTable: Record "NAV App Tenant Operation";
+                    ExtensionOperationImpl: Codeunit "Extension Operation Impl";
+                begin
+                    NavAppTenantOperationTable.CopyFilters(Rec);
+                    if NavAppTenantOperationTable.FindSet() then
+                        repeat
+                            ExtensionOperationImpl.RefreshStatus(NavAppTenantOperationTable."Operation ID");
+                        until NavAppTenantOperationTable.Next() = 0;
+                end;
+            }
+            action("Upload Extension")
```

Comment:
Everywhere else, we are adding the `ApplicationArea = All`
While it is not required, let's be consistent in this and add it here too.